tests/test_butler.py

```diff
@@ -262,11 +262,19 @@ class PosixDatastoreButlerTestCase(ButlerTests, lsst.utils.tests.TestCase):
     """PosixDatastore specialization of a butler"""
     configFile = os.path.join(TESTDIR, "config/basic/butler.yaml")
 
+    def testStringification(self):
+        butler = Butler(self.configFile)
+        self.assertEqual(str(butler), "Butler(collection='ingest', root='./butler_test_repository')")
```

Comment:
I'm not entirely sure I want this sort of hard-wired testing as I add more datastores and butler configs but maybe we have no choice.

Reply:
I've changed the tests to use `self.assertIn()`, with multiple asserts to test for the expected values.